dev/dev/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for dev project.
+
+Generated by 'django-admin startproject' using Django 1.9.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.9/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.9/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '0anpi!i8i5qy(81veb#i7gtmsbvk8xrum&d04a1ute-^6wenk#'
```

Comment:
In the future we'll make sure secret key isn't checked in. But this is just an informative post for now. Don't worry about it yet.